x-pack/plugins/security_solution/public/detections/components/rules/related_integrations/use_installed_integrations.tsx

```diff
@@ -29,6 +33,9 @@ export const useInstalledIntegrations = ({ packages }: UseInstalledIntegrationsA
       },
     ],
     async ({ signal }) => {
+      if (skip) {
+        return [];
+      }
```

Comment:
The `useQuery` hook from tanstack has an [`enabled` option flag](https://tanstack.com/query/v4/docs/react/guides/disabling-queries) that I believe accomplishes this same thing, any reason for not using that?